lib/import/helpers/colorectal/providers/rtd/rtd_constants.rb

```diff
@@ -0,0 +1,122 @@
+                                              'presymptomatic test' => :targeted_mutation,
+                                              'storage' => :full_screen,
+                                              'unknown / other' => :no_gentictestscope,
+                                              'unknown' => :no_gentictestscope,
+                                              'msi screen' => :no_gentictestscope,
+                                              'rna studies' => :no_gentictestscope }.freeze
+
+            PASS_THROUGH_FIELDS_COLO = %w[age authoriseddate
+                                          requesteddate
+                                          specimentype
+                                          providercode
+                                          consultantcode
+                                          servicereportidentifier].freeze
+            FIELD_NAME_MAPPINGS_COLO = { 'consultantcode' => 'practitionercode',
+                                         'ngs sample number' => 'servicereportidentifier' }.freeze
+
+            INVESTIGATION_CODE_GENE_MAPPING = {
+              'fap' => %w[APC],
+              'fap/map' => %w[APC MUTYH],
+              'fap-mlpa' => %w[APC],
+              'hnpcc' => %w[MLH1 MSH2 MSH6 PMS2 EPCAM],
+              'hnpcc_pred' => %w[MLH1 MSH2 MSH6 PMS2 EPCAM],
+              'hnpcc-mlpa' => %w[MLH1 MSH2 MSH6 EPCAM],
+              'colorectal cancer' => %w[APC BMPR1A EPCAM GREM1 MLH1 MSH2 MSH6 MUTYH NTHL1 PMS2
+                                        POLD1 POLE PTEN SMAD4 STK11],
+              'colerectal cancer' => %w[APC BMPR1A EPCAM GREM1 MLH1 MSH2 MSH6 MUTYH NTHL1 PMS2
+                                        POLD1 POLE PTEN SMAD4 STK11],
+              'tp53' => %w[TP53]
+            }.freeze
+
+            NON_PATHEGENIC_CODES = ['nmd',
+                                    'likely benign',
+                                    'benign',
+                                    'non-pathological variant'].freeze
+            # rubocop:disable Lint/MixedRegexpCaptureTypes
+            CDNA_REGEX = /c\.?,?\[?(?<cdna>
+                                ([0-9]+[+>_-][0-9][+>_-][0-9]+[+>_-][0-9][ACGTdelinsup]+)|
+                                ([0-9]+[+>_-][0-9][+>_-][0-9]+[+>_-][0-9]+[ACGTdelinsup]+)|
+                                ([0-9]+[+>_-][0-9]+[ACGTdelinsup][+>_-][ACGTdelinsup])|
+                                ([0-9]+[ACGTdelinsup]+[+>_-][ACGTdelinsup])|
+                                ([0-9]+[+>_-][0-9]+[ACGTdelinsup]+)|
+                                ([0-9]+[+>_-][0-9]+[ACGTdelinsup]?)|
+                                ([0-9]+[+>_-][0-9]+[+>_-][0-9]+[0-9]+[ACGTdelinsup]+)|
+                                ([0-9]+[?+>_-]+[0-9]+[?+>_-]+[ACGTdelinsup]+)|
+                                ([0-9]+[ACGTdelinsup]+)|
+                                ([+>_-][0-9]+[ACGTdelinsup>]+)
+                                )\]?/ix.freeze
```

Comment:
As of ruby 3.x, Rubocop will compain here:`Do not freeze immutable objects, as freezing them has no effect`

Reply:
done ✅